test/SqlExtension.Tests/SqlOutputBindingTests.cs

```diff
@@ -40,5 +40,41 @@ public async Task TestAddAsync()
             };
             await collector.AddAsync(data);
         }
+
+        [Fact]
+        public void TestGetTableAndSchema()
```

Comment:
Would suggest combining these all into a single test using theories and inline data